apps/docs/src/app/platform/component-docs/platform-forms/radio-group/platform-radio-group-docs.component.html

```diff
@@ -1,44 +1,46 @@
-<fd-docs-section-title [id]="'def'" [componentName]="'Radio Button Group'">
-    Radio Button Group with Radio buttons as Content
+<fd-docs-section-title id="projection" componentName="RadioGroup">
+    Radio Group With Radio Buttons Passed as Content Projection
 </fd-docs-section-title>
 
-<description>This example shows Radio Button Group implementaion with Content as Radio Buttons</description>
+<description>
+    Radio group is created from platform radio buttons. which is passed as content projection
```

Comment:
Almost!! Just need to change the placement of the "." and add a comma, here's the correction: "The radio group is created from platform radio buttons, which is passed as content projection."